tests/test_calculate_recurrent_infections.py

```diff
@@ -57,8 +59,9 @@ def test_infections_occur_not_in_other_recurrent_group():
     indexers.append(sub_indexers)
 
     infection_probabilities = np.array([1])
-    infected = np.array([False, True, False, False])
+    infected = np.array([False, False, False, False])
```

Comment:
Why did this change?

Reply:
infected indicates individuals which are newly infected. The former should be the outcome not the input.